app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @recipe.name %></h1>
+<h2>Complexity: <%= @recipe.complexity %></h2>
+<h2>Genre: <%= @recipe.genre %></h2>
+<% @recipe_ingredients.each do |recipe_ingredient| %>
+  <ul>
+    <%= recipe_ingredient.ingredient.name %>
+  </ul>
+<% end %>
+<h3>Total Cost: $<%= @recipe.total_cost %></h3>
+<br>
+<h3>Add Ingredient to <%= @recipe.name %></h3>
+<%= form_with url: "/recipes/#{@recipe.id}", method: :patch, local: true do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name%>
```

Comment:
Double-check the user story for extension # 2 - does it ask you to create a new recipe or ingredient? If so, this form setup would be fine. But it asks us to input an ingredient id, and that's all the form should need.